app/assets/javascripts/browsered/worldpay-3ds-flex-ddc.js

```diff
@@ -43,9 +43,11 @@ const checkOriginHostName = origin => {
 }
 
 const submitWithWorldpay3dsFlexDdcResult = form => {
+  const WORLDPAY_RESPONSE_TIMEOUT = 30000
```

Comment:
`DDC_TIMEOUT_IN_MILLISECONDS`? (DDC is more specific than Worldpay — most of the time we’ll be waiting for CardinalCommerce rather than Worldpay — and including the units is always good.)

Reply:
Agreed